server/testsupport/createInductionDtoTestDataBuilder.ts

```diff
@@ -0,0 +1,147 @@
+import type { CreateInductionDto } from 'dto'
+
+const aCreateLongQuestionSetInductionDto = (
+  hasWorkedBefore?: boolean,
```

Comment:
It's fine for now, but we would be better off with this being an optional `options` object which in turn contains these optional fields (see my test data builders)

I agree its a bit harder to grok, especially if you are relatively new to typescript, but in terms of the calls to these methods its much much better (and more akin to kotlin's named args 😁 )

The problem is typescript doesnt have named args. This method has 3 optional args, so a test can safely call `aCreateLongQuestionSetInductionDto()` with no args (which I suspect is what the tests are currently doing)

If however you wanted to call it with `hasInterests: false`, because you cant use named args, you'd have to specify all 3 args, but without any names (or context for readability):
```
aCreateLongQuestionSetInductionDto(null, null, false)
```

By contrast, if you use an optional `options` object, you effectively get named params. So the above call would be:
```
aCreateLongQuestionSetInductionDto({ hasInterests: false })
```
Much better 😁

Reply:
I copied the `options` from your builder, but I ended up only having `prisonId` in them, so it seemed somewhat unnecessary?

Reply:
After our discussion, I've switched to use the `options` approach.